lib/rx/client.rb

```diff
@@ -66,6 +73,11 @@ def post_preferences(params)
 
     private
 
+    def cache_key(action)
+      return nil unless CACHING_ENABLED
```

Comment:
FYA: There's a `FeatureFlipper` class that could use for this _sort_ of toggle, and drive it based of some user info, ENV, Setting, or whatever else.